### ISSUE TITLE
No way to set env tag

### ISSUE THREAD

**Issue by nirvdrum** [DataDog/browser-sdk#278]
All of our logs going into Datadog are tagged by environment (path name `env`). This tag is set up by the Datadog agents, so we don't really have control over the path. I followed the documentation on setting up global context for logging, which sets the `"env"` key:

```
datadogLogs.setLoggerGlobalContext({ env: 'custom-env-name', service: 'service-name' })
```

Unfortunately, this is coming into Datadog with path `@env`, rather than `env`. As such, the facet we're using to segregate logs by environment doesn't work. As far as I can tell, there's nothing exposed in the API that would allow setting up this tag.

**Comment by bcaudan** [comment]
Hi @nirvdrum!
For now, we don't have an API to send tags directly from the browser SDK.
However, if you are using logs, you can configure a [remapper](https://docs.datadoghq.com/logs/processing/processors/?tab=ui#remapper) to set an `env` tag from the `env` attribute.

**Comment by liss-mouse** [comment]
@bcaudan I have a similar question regarding `datadogRum`. How `addRumGlobalContext` is supposed to work? I can't find my tags in Datadog. Maybe I'm just missing something from the documentation?

**Comment by bcaudan** [comment]
Hi @liss-mouse,

You can find  `addRumGlobalContext` content in Datadog inside the events attributes.
From the explorer, inside the view side panel under the attributes tab.
From the analytics, by looking at a specific event.

**Comment by liss-mouse** [comment]
Thanks a lot, that helped 🙂 (DD interface is not that obvious for newbies 🙃)

**Comment by bcaudan** [comment]
There is on going work to improve the UI and the documentation on that front 🙂

**Comment by bcaudan** [comment]
env tag is available since v1.11.5 with https://github.com/DataDog/browser-sdk/pull/392
